integration-tests/testsetups/test_helpers.go

```diff
@@ -437,6 +455,12 @@ func CreateDockerEnv(t *testing.T) (
 			evmNetworks[i].URLs = rpcProvider.PrivateWsUrsl()
 		}
 		env.EVMNetworks = append(env.EVMNetworks, &evmNetworks[i])
+		// if number of private keys is more than 1, we can fund the nodes in parallel
```

Comment:
Oh I see, is this logic really required? Why not just always fund in parallel? Can't imagine it'd add that much overhead

Reply:
Do you mean change it to having >2 keys in private keys always and fail otherwise?